### ISSUE TITLE
Validation checkpoint/restore seems to cause convergence to falter

### ISSUE THREAD

**Issue by kdavis-mozilla** [mozilla/DeepSpeech#258]
From the current long run right before the first validation checkpoint/restore loss was about 88. After this checkpoint/restore the loss never dipped below 88 and it's currently with loss 112 at epoch 58. In addition the WER 0.72 at epoch 49 is far worse than that 0.57 at epoch 20.

**Comment by tilmankamp** [comment]
Maybe the optimizer looses some state during save/resume?

**Comment by kdavis-mozilla** [comment]
Originally I thought the optimizer did not loose state due to the comment[[1](http://bit.ly/2fJcxnw)]. But now, due to our current run, I am not sure.

**Comment by kdavis-mozilla** [comment]
It looks like setting `learning_rate=0.0001` solves this. Our previous `learning_rate` was simply to big.
